content/outcomes/secure-software-supply-chain/container-registry.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How a Container Registry Can Help
+weight: 6
+layout: single
+---
+
+A container registry provides a single place to store and retrieve your container images. At a minimum, a container registry will support the [Docker Registry API](https://docs.docker.com/registry/spec/api/), the API that both Docker and Kubernetes expect to be supported to pull container images. They may either be a publicly hosted such as [Docker Hub](https://hub.docker.com/) or hosted privately such as [VMware Harbor](https://goharbor.io/).
+
+## Automated Security Scans
+
+A popular feature of container registry is the ability to automatically scan container images for known vulnerabilities. As a single container image can container many layers, including a base container image that you don't even maintain, it can be a huge challenge to stay on top of keeping your dependencies up to date. Because of this, you'll find many container registries offer the ability to scan your containers every time you upload one and report if there's a known vulnerability discovered. For example, Harbor relies on [Clair](https://github.com/quay/clair), an open-source solution that performs static analysis on container images. Of course, as new vulnerabilities are discovered, you can rerun these scans against your existing images. If an issue is found, you can configure your registry to block any requests to pull the image based on the severity.
```

Comment:
I don't think harbor uses Clair by default anymore? I thought it was Trivy?

Reply:
Yup, you're right, looks like they changed that since I last looked into it. Good catch!